packages/aila/src/features/persistence/adaptors/file/index.ts

```diff
@@ -1,26 +1,28 @@
 import { AilaPersistence } from "../..";
-import type { AilaChatService, AilaServices } from "../../../../core/AilaServices";
+import type {
+  AilaChatService,
+  AilaServices,
+} from "../../../../core/AilaServices";
 import type { AilaGeneration } from "../../../generation/AilaGeneration";
 
 export class AilaFilePersistence extends AilaPersistence {
   constructor({ aila, chat }: { aila: AilaServices; chat: AilaChatService }) {
     super({ aila, chat, name: "AilaFilePersistence" });
   }
 
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
   async upsertChat(): Promise<void> {
     // Implementation goes here
-    throw new Error("Not implemented");
+    return Promise.reject(new Error("Not implemented"));
   }
```

Comment:
Was this creating a lint error? I can't see any difference, is it just that the lint ruls isn't smart enough to see that throwing in an async function returns a promise?

Reply:
I think so - it was showing as a linting error in VSCode for me and appearing in the Problems tab without this